src/features/form/sagas.js

```diff
@@ -96,11 +117,12 @@ function* submit(action) {
     let encounter = {};
     let updatedEncounter = {};
     let existingFlattenedObs = [];
+    let existingEncounter = action.encounter;
```

Comment:
Not strictly necessarily, but I did so it's clear what "action.encounter" is when we reference it later in the code, since I was confused myself at first...